prime-router/docs/how-to-onboard-a-receiver.md

```diff
@@ -192,109 +149,57 @@ If you want to create the data as HL7, that is easy to do as well:
+- Test again in Staging
+- If you are ready, carefully update settings in the prod environment. Especially in production, check the batch 
+timing. NOT every minute, eh?
 - If needed, push to production following our procedures for doing that.
 
-#### Testing locally
-The best way to test locally is to use the `quick-test.sh` shell script that we have in the `prime-router` folder in the project.
-
-```shell
-    # add your state here
-    for arg in "$@"
-    do
-      case "$arg" in
-        az | AZ) RUN_AZ=1;;
-        lt | LT) RUN_LT=1;;
-        all | ALL) RUN_ALL=1;;
-        merge | MERGE) RUN_MERGE=1;;
-      esac
-    done
-
-    # and add the state here as well
-    if [ $RUN_ALL -ne 0 ]
-    then
-      RUN_AZ=1
-      RUN_LT=1
-      RUN_STANDARD=1
-      RUN_MERGE=1
-    fi
-
-    # and then at a minimum run your state like this
-    # run LT
-    if [ $RUN_LT -ne 0 ]
-    then
-      LT_FILE_SEARCH_STR="/lt.*\.hl7"
-      echo Generate fake LT data
-      text=$(./prime data --input-fake 50 --input-schema lt/lt-covid-19 --output-dir $outputdir --target-states LT --output-format HL7_BATCH)
-      parse_prime_output_for_filename "$text" $LT_FILE_SEARCH_STR
-    fi
-```
-
-If you are generating CSV data, then you can test a roundtrip in the code:
-```shell
-    if [ $RUN_LT -ne 0 ]
-    then
-      LT_FILE_SEARCH_STR="/lt.*\.hl7"
-      echo Generate fake LT data
-      actual_lt=$(./prime data --input-fake 50 --input-schema lt/lt-covid-19 --output-dir $outputdir --target-states LT --output-format HL7_BATCH)
-      parse_prime_output_for_filename "$text" $LT_FILE_SEARCH_STR
-
-      # Now read the data back in to their own schema and export again.
-      # LT again
-      echo Test sending LT data into its own Schema:
-      text=$(./prime data --input-schema lt/lt-covid-19 --input $actual_lt --output-dir $outputdir)
-      parse_prime_output_for_filename "$text" $LT_FILE_SEARCH_STR
-      actual_lt2=$filename
-      compare_files "LT->LT" $actual_lt $actual_lt2
-    fi
-```
-
-Testing a round trip like this is good practice because it ensures your schema is valid in both directions and values are translating as you expect.
-
 ### Testing in your Docker container
 
-Another important step to take when onboarding a receiver is to start the docker container and then submit a file to the container and make sure that it translates and routes correctly.
+Another important step to take when onboarding a receiver is to start the docker container and then submit a file to 
+the container and make sure that it translates and routes correctly.
 
+Ensure that the transport is set to the default sftp:
+```
+transport:
+    type: SFTP
+    host: sftp
+    port: 22
+    filePath: ./upload
+    credentialName: DEFAULT-SFTP
+```
 First build the solution:
 
-`mvn clean package`
-
-Then build the docker image:
-
-`docker-compose build`
+`./gradlew clean package`
 
 Then start the docker container:
 
 `docker-compose up`
```

Comment:
Doesn't work properly on an Apple chip.

Reply:
@adegolier Do you know what does work? Feel free to make edits to this branch if you'd like

Reply:
You could just provide a direct link to the apple silicon instructions

https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/getting-started/Using-an-apple-silicon-mac.md

Reply:
added link